pkg/apis/kops/cluster.go

```diff
@@ -233,6 +233,33 @@ type ClusterSpec struct {
 
 	// Networking configuration
 	Networking *NetworkingSpec `json:"networking,omitempty"`
+
+	// API field controls how the API is exposed outside the cluster
+	API *PublishSpec `json:"api,omitempty"`
+}
+
+type PublishSpec struct {
```

Comment:
`NetworkingSpec` ----------------> The spec for which networking configuration to use
`TopologySpec` ------------------> The spec for which topology configuration to use
`KubernetesApiAccessSpec` -----> The spec for which Kubernetes API configuration to use


thoughts?